base/bucket_gocb.go

```diff
@@ -305,6 +306,7 @@ func (bucket *CouchbaseBucketGoCB) GetMaxTTL() (int, error) {
 		return -1, err
 	}
 	username, password, _ := bucket.spec.Auth.GetCredentials()
+	log.Printf("Not Setting basic auth: %q %q", username, password)
```

Comment:
Should this be an `Infof(KeyAll,..) statement`

Reply:
Thanks for catching - that should be removed altogether.